website/public/locales/en/index.json

```diff
@@ -6,7 +6,15 @@
     "q0": "How far along is this project?",
     "a0": "We are in the early stages of development, working from established research in applying RLHF to large language models.",
     "q1": "Who is behind Open Assistant?",
-    "a1": "Open Assistant is a project organized by LAION and individuals around the world interested in bringing this technology to everyone."
+    "a1": "Open Assistant is a project organized by LAION and individuals around the world interested in bringing this technology to everyone.",
+    "q2": "What license does Open Assistant use?",
+    "a2": "The code and models are licensed under the Apache 2.0 license.",
+    "q3": "Will the training data also be released?",
+    "a3": "Yes, also under Apache 2.0.",
+    "q4": "Will Open Assistant be free?",
+    "a4": "Yes, Open Assistant will be free to use and modify.",
+    "q5": "What hardware will be required to run the models?",
+    "a5": "There will be versions which will be runnable on consumer hardware. 8GB of VRAM."
```

Comment:
I would not say 8GB of ram, just consumer hardware. we don't know any of this yet

Reply:
👍